### PR TITLE
URL and mailmap updates, remove an obsolete directory

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

File: lib/socks.c [hunk 1]
```diff
@@ -98,7 +98,7 @@ int Curl_blockread_all(struct connectdata *conn, /* connection data */
 * destination server.
 *
 * Reference :
-*   http://socks.permeo.com/protocol/socks4.protocol
+*   https://web.archive.org/web/20070102104129/socks.permeo.com/protocol/socks4.protocol
```
Comment (bagder) [inline lib/socks.c]: I propose using https://www.openssh.com/txt/socks4.protocol instead

File: packages/EPM/README [hunk 2]
```diff
@@ -8,5 +8,4 @@ archives from a list of files. EPM Can:
  * Provide a complete, cross-platform software distribution solution for your
    applications.
 
-http://www.easysw.com/epm/
-
+https://web.archive.org/web/20071225231927/www.easysw.com/epm/
```
Comment (bagder) [inline packages/EPM/README]: maybe this is a subtle hint we can just as well remove this entire dir?

File: src/Makefile.netware [hunk 3]
```diff
@@ -5,7 +5,7 @@
 #                            | (__| |_| |  _ <| |___
 #                             \___|\___/|_| \_\_____|
 #
-# Copyright (C) 2004 - 2014, Guenter Knauf, <http://www.gknw.net/phpbb>.
+# Copyright (C) 2004 - 2014, Guenter Knauf, <https://web.archive.org/web/20100126042636/www.gknw.net/phpbb>.
```
Comment (bagder) [inline src/Makefile.netware]: here again

**Comment by vszakats** [comment]
@bagder: Good points, thanks! Changes committed.

**Comment by bagder** [comment]
lgtm seems deeply confused =)

**Comment by vszakats** [comment]
LGTM had a fair point: EPM was referenced from a few places, which I now removed along with the directory itself. Let's see how it goes.

**Pull request closed without merging**